### PR TITLE
Fix typo and add entry point

### PR DESCRIPTION
**Pull request by innovate-invent**
Sorry, I missed this typo

### CODE REVIEW

File: src/ephemeris/install_tool_deps.py [hunk 1]
```diff
@@ -14,8 +14,8 @@
 def _parser():
     parent = get_common_args()
     parser = argparse.ArgumentParser(parents=[parent])
-    parser.add_argument("-t", "--tool", help='Path to a tool file, tool_conf file, or yaml file containing sequence of tool ids', nargs='*')
-    parser.add_argument("-i", "--id", help='Comma seperated list of tool ids', nargs='*')
+    parser.add_argument("-t", "--tool", help='Path to a tool file, tool_conf file, or yaml file containing a sequence of tool ids', nargs='*')
+    parser.add_argument("-i", "--id", help='Space separated list of tool ids', nargs='*')
```
Comment (nsoranzo) [inline src/ephemeris/install_tool_deps.py]:
```suggestion
    parser.add_argument("-i", "--id", help='Space-separated list of tool ids', nargs='*')
```

Comment (innovate-invent) [reply src/ephemeris/install_tool_deps.py]: I have no objection to this suggestion

File: src/ephemeris/install_tool_deps.py [hunk 2]
```diff
@@ -28,29 +32,71 @@ def main():
...
+                                tool_client.install_dependencies(tool_id)
+                            except ConnErr as e:
+                                if e.status_code in timeout_codes:
+                                    log.warning(e.body)
+                                else:
+                                    raise
+                elif root.tag == "tool" and root.get('id'):
+                    # Install from single tool file
+                    log.info("Tool xml found. Installing " + root.get('id') + " dependencies..")
+                    try:
```
Comment (mvdbeek) [inline src/ephemeris/install_tool_deps.py]: can you move the try/except handling to a separate function ?

**Comment by innovate-invent** [comment]
CI failure is unrelated

**Comment by mvdbeek** [comment]
Thanks @innovate-invent !

**Pull request merged**